app/controllers/kaui/bundles_controller.rb

```diff
@@ -6,16 +6,28 @@ def index
       all_bundle_tags = @account.all_tags(:BUNDLE, false, 'NONE', options_for_klient)
       all_bundle_tags.inject({}) {|hsh, entry| (hsh[entry.object_id] ||= []) << entry; hsh}
     }
+    fetch_subscription_tags = promise {
+      all_subscription_tags = @account.all_tags(:SUBSCRIPTION, false, 'NONE', options_for_klient)
+      all_subscription_tags.inject({}) {|hsh, entry| (hsh[entry.object_id] ||= []) << entry; hsh}
+    }
     fetch_bundle_fields = promise {
       all_bundle_fields = @account.all_custom_fields(:BUNDLE, 'NONE', options_for_klient)
       all_bundle_fields.inject({}) {|hsh, entry| (hsh[entry.object_id] ||= []) << entry; hsh}
     }
+    fetch_subscription_fields = promise {
+      all_subscription_fields = @account.all_custom_fields(:SUBSCRIPTION, 'NONE', options_for_klient)
+      all_subscription_fields.inject({}) {|hsh, entry| (hsh[entry.object_id] ||= []) << entry; hsh}
+    }
     fetch_available_tags = promise { Kaui::TagDefinition.all_for_bundle(options_for_klient) }
+    fetch_available_subscription_tags = promise { Kaui::TagDefinition.all_for_subscription(options_for_klient) }
```

Comment:
Can we make sure to get rid of internal system tags in these lists (see https://github.com/killbill/killbill/blob/7120cfd1bf4861489f2ee46f8b9fc6231bacf877/util/src/main/java/org/killbill/billing/util/tag/api/DefaultTagUserApi.java#L132)?

It's an existing bug but might as well fix it as part of this PR because it could affect CSS (e.g. by default, no tag definition is applicable on subscriptions, so the dropdown would be empty).